.travis.yml

```diff
@@ -1,14 +1,16 @@
 sudo: required
-dist: trusty
+dist: xenial
 language: cpp
 
 before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
-  - sudo apt-get install -qq g++-4.8 libarmadillo-dev libarmadillo4 libboost1.54-all-dev
   - export CC=gcc-4.8
   - export CXX=g++-4.8
   - printenv
+  - sudo apt-get install libopenblas-dev
+  - wget http://sourceforge.net/projects/arma/files/armadillo-7.950.1.tar.xz && tar xf armadillo-7.950.1.tar.xz
```

Comment:
@Saurabh3012 @rcurtin If I remember right you work with some older armadillo versions, maybe we can use masterblaster to provide the package?